web/src/engine/osk/src/banner/bannerView.ts

```diff
@@ -245,18 +253,27 @@ export class BannerController {
    *        representing the type of `Banner` to set active.
    * @param height - Optional banner height in pixels.
    */
-  public setBanner(type: BannerType, height?: number) {
+  public setBanner(type: BannerType) {
```

Comment:
Is this an API change?

Reply:
The parameter was literally never used, and this function itself has never been documented as API.